code/datums/skills/_skill.dm

```diff
@@ -5,6 +5,7 @@
 	var/modifiers = list(SKILL_SPEED_MODIFIER = list(1, 1, 1, 1, 1, 1, 1)) //Dictionary of modifier type - list of modifiers (indexed by level). 7 entries in each list for all 7 skill levels.
 	var/skillcapepath
 	var/list/levelUpMessages = list()
+	var/list/levelDnMessages = list()
```

Comment:
dont shorten var names like that unepic ;(

Reply:
youre unepic fuck you

Reply:
![image](https://user-images.githubusercontent.com/29008542/78377572-c4c85500-759d-11ea-9388-ecaa417ecf47.png)